fastlane/Helpers/Build.swift

```diff
@@ -26,7 +26,7 @@ enum Build {
             outputDirectory: Constant.outputPath,
             outputName: .userDefined(environment.productName),
             includeSymbols: .userDefined(true),
-            includeBitcode: .userDefined(type == .appStore),
+            includeBitcode: .userDefined(false),
```

Comment:
What do you think about remove this property as it should always be `false` (default is `nil`), so that no one misuse it as true?

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/376/commits/64a95453621ed3b5d4c826a71897affede18a32a